src/Analysis/GWASV2/Components/SelectConfiguration/SelectConfiguration.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SelectConfiguration from './SelectConfiguration';
+import ValidInitialState from '../../TestData/InitialStates/ValidInitialState';
+import ACTIONS from '../../Utils/StateManagement/Actions';
+
+describe('SelectConfiguration component', () => {
+  let wrapper;
+  const dispatch = jest.fn();
+  const {
+    numOfPCs,
+    mafThreshold,
+    selectedStudyPopulationCohort,
+    covariates,
+    outcome,
+    imputationScore,
+  } = ValidInitialState;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <SelectConfiguration
+        numOfPCs={numOfPCs}
+        mafThreshold={mafThreshold}
+        selectedCohort={selectedStudyPopulationCohort}
+        covariates={covariates}
+        outcome={outcome}
+        dispatch={dispatch}
+        imputationScore={imputationScore}
+      />,
+    );
+  });
+  it('should render the InputNumber component for the number of PCs', () => {
```

Comment:
The test at line 44 already covers this, no?

Reply:
That is true, but I believe testing for something existing, and then later testing its functionality is a common pattern, especially in TDD.

Reply:
Per our discussion: this has been removed now.